node-src/index.ts

```diff
@@ -24,7 +24,13 @@ interface Output {
   inheritedCaptureCount: number;
 }
 
-export async function runChromatic(flags: Flags): Promise<Output> {
+export async function runChromaticFull({
+  flags = {},
+  options = {},
+}: {
+  flags?: Flags;
+  options?: Partial<Options>;
+}): Promise<Output> {
```

Comment:
I'm not digging the name `runChromaticFull`. What does "full" mean in this context? I think we should treat this node entry point as a public API for anyone who wants to integrate more deeply with our CLI. I think it should either be a descriptive and truthful name, or be something quite generic like `run` or `main`.